src/lib/mongoBackend/MongoGlobal.h

```diff
@@ -150,6 +150,14 @@ using namespace mongo;
 
 /* ****************************************************************************
 *
+* OtisTreatFunction - callback signature for treatOntimeintervalSubscriptions
+*/
+typedef void (*OtisTreatFunction)(std::string tenant, BSONObj& bobj);
```

Comment:
The new one is called (after renaming it): *subToCache*.
Two older ones are:
  - recoverOnTimeIntervalThread
  - destroyOnTimeIntervalThread

Before this type was used only inside MongoGlobal.cpp and the type was there too, of course.
Now that it is used also from SubscriptionCache.cpp, the type had to be moved to a header file.

NTC